### PR TITLE
Navigation links updates

### LINKED ISSUE #6114: Add Coronavirus links to Navigation
**Is your feature request related to a problem? Please describe.**
Add a link for Coronavirus to the sites below
The link should always appear in the SCOND position

**Describe the solution you'd like**
For all sites add the item name: coronavirus to the Second positon in the followinig sites

Hindi
title : 'कोरोनावायरस'
url : '/hindi/international-51848794'

Ukraine
title : 'Коронавірус'
url : '/ukrainian/52118881'

Japanese
title : 'コロナウイルス'
url : '/japanese/52137815'

Serbian/lat
title : 'Korona virus'
url : '/serbian/lat/srbija-52197807'

Serbian/cyr
title : 'Корона вирус'
url : '/serbian/cyr/srbija-52197807'

**Describe alternatives you've considered**
N/A

**Testing notes**
Updates navigation on live pages for Hindi, Ukrainian, Japanese, Serbian/lat Serbian/cyr

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #6114 #6115 #6116 #6117 
**PAL goes live with these changes on Tuesday 14/04/2020**

**Overall change:**
Updated navigation links in various services.

**Code changes:**
- Updated links in service configs for Hindi, Japanese, Nepali, Persian, Ukranian

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8026,7 +8026,7 @@
     },
     "diffable-html": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/diffable-html/-/diffable-html-4.0.0.tgz",
```
Comment (OlgaLyubin) [inline package-lock.json]: All navigation changes look good, but `package-lock.json` should not be updated?

Comment (hotinglok) [reply package-lock.json]: No idea how that happened, but fixed that now.
isoladeifamosi-isola-isola14-isola2019-1Y64pa5uJW5nKMjYDm

**Review by OlgaLyubin — APPROVED** [review]
Thanks for the update, LGTM 👍

**Review by Bopchy — APPROVED** [review]
Looks great!

**Comment by PriyaKR** [comment]
LGTM.

**Comment by sareh** [comment]
Can now be merged, after CI passes.

**Pull request merged**